guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -203,12 +209,8 @@ This image is generic to all hosts with a provisioning NIC on the same subnet.
 For more information about configuring security tokens, read xref:Provisioning_Bare_Metal_Hosts-Security_Token_in_the_Boot_Process[].
 
 [NOTE]
-ifdef::satellite[]
-The *Full host image* is based on SYSLINUX and works with all {RHEL} certified hardware.
-endif::[]
-ifndef::satellite[]
 The *Full host image* is based on SYSLINUX and works with most hardware.
-endif::[]
+
 When using a *Host image*, *Generic image*, or *Subnet image*, see https://ipxe.org/appnote/hardware_drivers[supported hardware on ipxe.org] for a list of hardware drivers expected to work with an iPXE-based boot disk.
```

Comment:
For satellite context, you need to render this sentence differently:

    When using a *Subnet image*, see https://ipxe.org/appnote/hardware_drivers[supported hardware on ipxe.org] for a list of hardware drivers expected to work with an iPXE-based boot disk.